src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -11,32 +11,34 @@
     "Microsoft.Build": "0.1.0-preview-00017",
     "Microsoft.Net.Compilers.NetCore": "1.2.0-beta1-20160122-02",
     "Microsoft.Net.Compilers.Targets.NetCore": "0.1.4-dev",
-    "Microsoft.NETCore.TestHost": "1.0.0-rc3-23811",
-    "Microsoft.NETCore.Console": "1.0.0-rc3-23811",
-    "Microsoft.NETCore.Platforms": "1.0.1-rc3-23811",
-    "Microsoft.NETCore.Runtime": "1.0.1-rc3-23811",
+    "Microsoft.NETCore.TestHost": "1.0.0-rc2-23816",
+    "Microsoft.NETCore.Console": "1.0.0-rc2-23816",
+    "Microsoft.NETCore.Platforms": "1.0.1-rc2-23816",
+    "Microsoft.NETCore.Runtime": "1.0.1-rc2-23816",
     "Microsoft.Cci": "4.0.0-rc3-23811",
     "MSBuild": "0.1.0-preview-00017",
     "Newtonsoft.Json": "7.0.1",
     "NuGet.Versioning": "3.4.0-beta-488",
     "NuGet.Client": "3.4.0-beta-488",
-    "System.IO.Compression": "4.1.0-rc3-23811",
-    "System.Dynamic.Runtime": "4.0.11-rc3-23811",
-    "System.Linq.Expressions": "4.0.11-rc3-23811",
-    "System.Diagnostics.TextWriterTraceListener": "4.0.0-rc3-23811"
+    "System.IO.Compression": "4.1.0-rc2-23816",
+    "System.Dynamic.Runtime": "4.0.11-rc2-23816",
+    "System.Linq.Expressions": "4.0.11-rc2-23816",
+    "System.Diagnostics.TextWriterTraceListener": "4.0.0-rc2-23816",
+    "System.IO.FileSystem": "4.0.1-rc2-23816",
```

Comment:
@ericstj when you are back I want to better understand why we needed to directly reference these in order to get the correct runtime packages. Without directly referencing FileSystem and Algorithms we were getting the correct contract package but we were getting older runtime packages for those guys. The older runtime packages were comming from the MSBuild and Compiler package references but I cannot understand why it picked a mis-match between the contract package and runtime package versions.